wof/core_1_0.py

```diff
@@ -5,8 +5,8 @@
 
 import logging
 
-import core
-import WaterML
+import wof.core as core
```

Comment:
It is more common to use `from wof import core` but I don't have a preference. Feel free to use whatever you think it is best.